com.unity.render-pipelines.universal/Shaders/SpatialMappingOcclusion.shader

```diff
@@ -0,0 +1,72 @@
+Shader "Universal Render Pipeline/SpatialMappingOcclusion"
+{
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "IgnoreProjector" = "False" "RenderPipeline" = "UniversalPipeline" }
+        LOD 100
+
+        Blend One OneMinusSrcAlpha
+        ZWrite On
+        ZTest LEqual
+        Colormask 0
+        Cull Off
+
+        Pass
+        {
+            Name "Spatial Mapping Occlusion"
+
+            HLSLPROGRAM
+            #pragma gles gles3 glcore
+            #pragma target 2.0
```

Comment:
Both of these pragmas can be fully removed, looking at the original shader, it has no target or renderers set